readthedocs/core/models.py

```diff
@@ -43,6 +43,13 @@ class UserProfile(TimeStampedModel):
     whitelisted = models.BooleanField(_("Whitelisted"), default=False)
     banned = models.BooleanField(_("Banned"), default=False)
 
+    # Opt-out on emails
+    # NOTE: this is a temporary field that we can remove after September 25, 2023
+    # See https://blog.readthedocs.com/migrate-configuration-v2/
+    optout_email_config_file_deprecation = models.BooleanField(
+        _("Opt-out from email about 'Config file deprecation'."), default=False
```

Comment:
I think this will break for profile lookups during deploy because it isn't nullable, just as a note. Not sure how wide ranging that will be, but might break a lot 🤷

Reply:
Good point! Done.